DER/DerivativesContracts/Options.rdf

```diff
@@ -489,47 +417,15 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Such terms may relate to the underlying instruments, exercise price, expiration date, and contract size, for example.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-der-opt;StrikePrice">
+	<owl:Class rdf:about="&fibo-der-drc-opt;StrikePrice">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
 		<rdfs:label xml:lang="en">strike price</rdfs:label>
-		<skos:definition xml:lang="en">The strike price for the option is the agreed price for the transaction if this takes place. Applies to all options types. Action: Make sure that&apos;s there for all Options types.</skos:definition>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-der-der-opt;TradedOptionPrincipal">
-		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;OptionIssuer"/>
-		<rdfs:subClassOf rdf:resource="&fibo-fbc-fct-mkt;ExchangeParticipant"/>
-		<rdfs:label xml:lang="en">traded option principal</rdfs:label>
-		<owl:disjointWith rdf:resource="&fibo-der-der-opt;TradedOptionsCounterparty"/>
-		<skos:definition xml:lang="en">The principal to the Traded Option contract. THis is the party which gives the other party the right to take up the obligation, and which is obliged to honor that transaction if that option is taken up.</skos:definition>
+		<skos:definition xml:lang="en">price at which an underlying asset may be bought or sold when the contract is exercised</skos:definition>
```

Comment:
strictly speaking it's a lot of the underlying asset (e.g. 100 shares)

Reply:
@rivettp - Fixed